### PR TITLE
Added tales-of-majeyal again and updated to 1.6.0

### PR DESCRIPTION
**Pull request by docquantum**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/tales-of-majeyal.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'tales-of-majeyal' do
+  version '1.6.0'
+  sha256 'c7ca6ecc1f6a0220bc4a5f27de99a988085ba997ccf406c657051f3df9805a0c'
+
+  url "https://te4.org/dl/t-engine/t-engine4-osx-#{version}.zip"
+  appcast 'https://te4.org/download'
+  name 'Tales of Maj\'Eyal'
```
Comment (reitermarkus) [inline Casks/tales-of-majeyal.rb]: Use double quotes to avoid escaping the single quote.

Comment (docquantum) [reply Casks/tales-of-majeyal.rb]: Good suggestion. Fixed now.

This is based on previous work, but the original maintainer stopped, so I pulled to file and updated it to the newest version of this game.

**Comment by reitermarkus** [comment]
@vitorgalvao, not sure if it should be `tales-of-majeyal` or `tales-of-maj-eyal`.

**Comment by docquantum** [comment]
From my experience, people normally search for it without the apostrophe, but that's up to you all.

As another note, some people also reference it as TOME for short.

**Comment by reitermarkus** [comment]
I guess e.g. “don't” should be `dont` instead of `don-t`, so I think this is fine as is.

**Comment by vitorgalvao** [comment]
Somehow missed this. Looks like I’m not getting emails when I’m @mentioned; will have to look into that.

I agree with ignoring apostrophes.

**Pull request merged**